microcosm_flask/conventions/crud.py

```diff
@@ -153,10 +154,14 @@ def configure_retrieve(self, ns, definition):
         :param definition: the endpoint definition
 
         """
+        request_schema = definition.request_schema or Schema()
+
         @self.graph.route(ns.instance_path, Operation.Retrieve, ns)
+        @qs(request_schema)
```

Comment:
Minor concern: does registering an empty query string schema cause any existing retrieve definitions to change? Or is this a noop?